pymc3/math.py

```diff
@@ -474,3 +474,23 @@ def block_diagonal(matrices, sparse=False, format="csr"):
     if len(matrices) == 1:  # graph optimization
         return matrices[0]
     return BlockDiagonalMatrix(sparse=sparse, format=format)(*matrices)
+
+
+def _linear_training_data():
+    r"""Create a simple linear model training dataset
+    for use in the documentation
+
+    Returns
+    -------
+    X: np.ndarray()
+        Array of input values
+
+    y: np.ndarray()
+        Array of output values
+
+    """
+    np.random.seed(1234)
```

Comment:
this should use the new generator, otherwise it will globally set the seed of numpy.random behind the scenes and users won't get random behaviour when they expect it to. https://numpy.org/doc/stable/reference/random/index.html#quick-start

Reply:
Done! I had no idea that was update. Thanks for letting me know